qgcimages.qrc

```diff
@@ -149,6 +149,15 @@
         <file alias="PX4/BrandImage">src/FirmwarePlugin/PX4/PX4BrandImage.png</file>
         <file alias="Quad.svg">src/ui/toolbar/Images/Quad.svg</file>
         <file alias="RadioComponentIcon.png">src/AutoPilotPlugins/Common/Images/RadioComponentIcon.png</file>
+        <file alias="RID_EMERGENCY_BACKGROUND_HIGHLIGHT_SVG.svg">src/ui/toolbar/Images/RID_EMERGENCY_BACKGROUND_HIGHLIGHT_SVG.svg</file>
```

Comment:
Can you stick to the existing convention?
```suggestion
        <file alias="RidEmergencyBackgroundHighlight.svg">src/ui/toolbar/Images/RidEmergencyBackgroundHighlight.svg</file>
```

Reply:
You are totally right, I am sorry. It should be fixed now